XRDXRFutils/gammasearch.py

```diff
@@ -148,35 +192,28 @@ def MSEloss(self):
         return array([phase_search.MSEloss() for phase_search in self]).reshape(self.shape)
 
     def selected(self):
-        return array([phase_search.idx for phase_search in self]).reshape(self.shape)
+        return array([phase_search.idx for phase_search in self]).reshape((self.shape[0], self.shape[1]))
```

Comment:
why is this different?

Reply:
> why is this different?

Because all the other functions return a numpy array with shape like (95, 170, 5), where last dimension is the number of phases. On the other hand, this function `selected()` returned a numpy array with shape (95, 170, 1), which we had to `squeeze()` before using. With this fix, the function returns a shape of (95, 170) as we need.